src/Extension/Entity/Extension.php

```diff
@@ -0,0 +1,77 @@
+<?php declare(strict_types=1);
+
+namespace OAT\DependencyResolver\Extension\Entity;
+
+class Extension
+{
+    const DEFAULT_BRANCH = 'develop';
+    const BRANCH_PREFIX = 'dev';
+
+    /** @var string */
+    private $extensionName;
+
+    /** @var string */
+    private $repositoryName;
+
+    /** @var string */
+    private $composerName = '';
+
+    /** @var string */
+    private $branchName = self::DEFAULT_BRANCH;
```

Comment:
This default value is useless because of constructor

Reply:
Removed